api/src/download.py

```diff
@@ -100,7 +40,15 @@ def download():
             mod_row.pop(id_idx)
             writer.writerow(mod_row)
 
-    response = send_file(f"./output/{filename}", mimetype = 'text/csv', attachment_filename = filename, as_attachment=True)
-    response.headers.add('Access-Control-Allow-Origin', '*')
+    file_response = send_file(f"./output/{filename}", mimetype = 'text/csv', attachment_filename = filename, as_attachment=True)
```

Comment:
Change `attachment_filename` to be consistent with our naming schema as described in other comment

Reply:
fixed